rmgpy/cantherm/explorer.py

```diff
@@ -0,0 +1,232 @@
+                                          bimolecularReact=np.zeros((len(reaction_model.core.species),len(reaction_model.core.species))))
+                    
+                        flags = np.array([s.molecule[0].getFormula()==form for s in reaction_model.core.species])
+                        reaction_model.enlarge(reactEdge=True,unimolecularReact=flags,
+                                          bimolecularReact=np.zeros((len(reaction_model.core.species),len(reaction_model.core.species))))
+                        incomplete = True
+        
+        #clean up output files
+        if outputFile is not None:
+            path = os.path.join(reaction_model.pressureDependence.outputFile,'pdep')
+            for name in os.listdir(path):
+                if name.endswith('.py') and '_' in name:
+                    if name.split('_')[-1].split('.')[0] != str(len(network.isomers)):
+                        os.remove(os.path.join(path,name))
+                    else:
+                        os.rename(os.path.join(path,name),os.path.join(path,'network_full.py'))    
+        
+        warns = []
+        
+        for rxn in jobRxns:
+            if rxn not in network.pathReactions:
+                warns.append('Reaction {0} in the input file was not explored during network expansion and was not included in the full network.  This is likely because your explore_tol value is too high.'.format(rxn))
+        
+        #reduction process
+        
+        if self.energy_tol != np.inf or self.flux_tol != 0.0:
+            
+            rxnSet = None
+            
+            for T in Tlist:
+                if self.energy_tol != np.inf:
+                    rxns = network.get_energy_filtered_reactions(T,self.energy_tol)
+                    if rxnSet is not None:
+                        rxnSet &= set(rxns)
+                    else:
+                        rxnSet = set(rxns)
+                    
+                for P in Plist:
+                    if self.flux_tol != 0.0:
+                        rxns = network.get_rate_filtered_reactions(T,P,self.flux_tol)
+                        if rxnSet is not None:
+                            rxnSet &= set(rxns)
+                        else:
+                            rxnSet = set(rxns) 
+                            
+            network.remove_reactions(reaction_model,list(rxnSet))
+        
+            for rxn in jobRxns:
+                if rxn not in network.pathReactions:
+                    warns.append('Reaction {0} in the input file was not included in the reduced model.'.format(rxn))
+    
+        self.network = network
+        
+        self.pdepjob.network = network
+        
+        self.pdepjob.execute(outputFile, plot, format='pdf', print_summary=True)
+        
+        if warns != []:
+            logging.info('\nOUTPUT WARNINGS:\n')
+            for w in warns:
+                logging.warning(w)
+
+
+        
+        
```

Comment:
The EOF still looks strange. Can you remove the tabs/spaces and leave just one empty line at the end?